packages/ui/helpers/utils/compact.js

```diff
@@ -0,0 +1,27 @@
+const Filters = {
+  "falsy": (value) => (value !== "") && (value !== undefined) && (value !== null) && (value !== 0),
+  "nullish": (value) => (value !== undefined) && (value !== null),
+};
+
+
+export function compact(collection, hasValue = Filters.falsy) {
+  if (Array.isArray(collection)) {
+    return collection.filter(hasValue);
+  }
+
+  if (typeof collection === "object") {
```

Comment:
if `collection` is null it will crash, right?

Reply:
👍 indeed. I'll update.